### DISCUSSION TITLE
How to persist data on reload . Headless-react

### DISCUSSION THREAD

**Discussion by vacool6** [coveo/ui-kit#2990] [Q&A]
Its possible to persist data when the app reloads / Is there a logic to persist data on reload.
As this is the default behavior of single-page apps do we need to implement the logic from scratch / Is there any custom logic available for it?

**Comment by louis-bompart** [comment]
> Its possible to persist data when the app reloads / Is there a logic to persist data on reload. As this is the default behavior of single-page apps do we need to implement the logic from scratch / Is there any custom logic available for it?

Hi, we do not offer such utils.
